homeassistant/components/bmw_connected_drive/config_flow.py

```diff
@@ -32,21 +33,21 @@ async def validate_input(
     Data has the keys from DATA_SCHEMA with values provided by the user.
     """
     try:
-        await hass.async_add_executor_job(
-            ConnectedDriveAccount,
+        account = MyBMWAccount(
```

Comment:
Please only wrap the line that can raise in the try... except block. If this doesn't do I/O we can move it up.